client/src/components/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react';
+import Header from "./Header";
+import SearchField from "./SearchField";
+import Footer from "./Footer";
+import Suggestion from "./data";
+
+class SearchPage extends Component{
```

Comment:
If `SearchPage` has no state of its own and is this simple, use a functional component instead of a class based one for it.